ci/tests/e2e.sh

```diff
@@ -59,6 +59,7 @@ mkdir -p /tmp/ci/sonobuoy
 
 curl -sfL -o /tmp/ci/sonobuoy/sonobuoy.yaml \
       https://raw.githubusercontent.com/heptio/sonobuoy/release-0.10/examples/quickstart.yaml
+sed -i -e 's/sonobuoy:master/sonobuoy:v0.10.0/g' /tmp/ci/sonobuoy/sonobuoy.yaml
```

Comment:
@timothysc Is this something that should be patched up in the quickstart.yaml file for the tagged version of sonobuoy? I'm not sure of the applicability of fixing this outside of our CI use.

Reply:
It's only really useful for CI right now, that patch will work.  Part of why I want to prevent hosting .yaml files in the repo that aren't auto-generated b/c they are out of date all the time.

Reply:
I'll try to patch this shortly...

Reply:
Use this URL - https://raw.githubusercontent.com/heptio/sonobuoy/v0.10.0/examples/quickstart.yaml